support-frontend-static/cfn.yaml

```diff
@@ -4,9 +4,7 @@ Resources:
   S3Bucket:
     Type: AWS::S3::Bucket
     Properties:
-      AccessControl: PublicRead
+      AccessControl: Private
```

Comment:
private actually means "no public list permission on the bucket, but individual get access depends on the object"